.github/workflows/docs.yaml

```diff
@@ -0,0 +1,76 @@
+on:
+  push:
+    branches:
+      - docs-workflow
```

Comment:
We need to change this to `master` when we merge this.

Reply:
Right! I used it to be able to test on my fork!